src/Menu/Menu.js

```diff
@@ -26,6 +33,127 @@ export default function Menu(props) {
         );
     });
 
+    const mainMenu = (
+        <div
+            className={cs("mainMenu", {
+                hidden: gameData.isMenuHidden,
```

Comment:
I don't think we need the "hidden" class anymore on anything because it's being hidden with conditional rendering and not with CSS